lib/WP_Auth0_Api_Client.php

```diff
@@ -2,6 +2,58 @@
 
 class WP_Auth0_Api_Client {
 
+	/**
+	 * Generate the API endpoint with a provided domain
+	 *
+	 * @since 3.4.1
+	 *
+	 * @param string $path - API path appended to the domain
+	 * @param string $domain - domain to use, blank uses default
+	 *
+	 * @return string
+	 */
+	public static function get_endpoint( $path = '', $domain = '' ) {
```

Comment:
@lbalmaceda - Everything is `public static` in that file, for better or worse. This is a case of consistency vs accuracy :)

Reply:
But if this file is going to end up being the "API Client" then I would prefer not to leave this method open for users to use, as it's only used internally, and also because once we publish this we can't remove it without breaking the public api.

Reply:
@lbalmaceda - I see what you're getting at here. Was a little confused with who you were referring to as "users" but I see the 💡